books/free-programming-books-langs.md

```diff
@@ -914,6 +914,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Airbnb CSS-in-JavaScript Style Guide](https://airbnb.io/javascript/css-in-javascript/) - Airbnb
 * [An advanced guide to HTML&CSS](http://learn.shayhowe.com/advanced-html-css/)
 * [Atomic Design](https://atomicdesign.bradfrost.com) - Brad Frost
+* [Building iPhone Apps with HTML, CSS, and JavaScript](https://www.dbooks.org/building-iphone-apps-with-html-css-and-javascript-0596805780/) -  Jonathan Stark(PDF, EPUB, Kindle)
```

Comment:
This appears not to be an authorized distribution. Please see CONTRIBUTING.md guidelines.